_assets/scss/elements/_typography.scss

```diff
@@ -78,6 +78,17 @@
   @include breakpoint($large) {}
 }
 
+@mixin paragraph {
```

Comment:
it would make sense, as long as H4 headlines and paragraphs are always at the same point of the type ramp